cypress/e2e/tests/migration/issues/filter.test.ts

```diff
@@ -25,11 +25,17 @@ import {
 import { Analysis } from "../../../models/migration/applicationinventory/analysis";
 import { SEC, filterIssue } from "../../../types/constants";
 import { Issues } from "../../../models/migration/issues/issues";
+import { BusinessServices } from "../../../models/migration/controls/businessservices";
```

Comment:
This might be a good time to move this test under "AllIssues" folder .

Reply:
Done